docs/index.md

```diff
@@ -3,12 +3,146 @@
+Plot points are converted automatically to the image's geographic coordinate system.
+
+### Sample data
+
+We also provide an [archive of sample data](https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz
+) [![external link](img/external_link.png)](https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz) that can be downloaded and used in this Tutorial.
+Be sure to extract the files from this archive before uploading them.
+Uploaded archives are **not** usable by the system.
+
+The following `bash` commands can be used to download the archive and extract the files.
+
+```bash
+curl --output sample_website_data.tar.gz https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz
+tar -xzf sample_website_data.tar.gz
+```
+
+### Uploading and configuring files
+
+On the main page of the site, click the `files` link under **Data Sources**.
+This will take you to the file storage definition page
+
+<img src="img/landing_click_files.png" width="450" alt="File storage definition link" aria-label="File storage definition link" title="Click on Files link"/>
+
+Next click the upload icon, select and upload the files to process.
+You may see an <img src="img/configure_files_uploading_files.png" width="90" alt="Uploading files" aria-label="Uploading files" title="Uploading files"/> message on the icon if it takes a few seconds to upload the files.
+You can also drag-and-drop files onto the upload location.
+
+<img src="img/configure_files_upload.png" width="450" alt="Click upload files icon" aria-label="Upload files link" title="Click or drag-and-drop to upload files"/>
+
+Next, select "Server Side" from the drop down on the upper right of the page, and click "New".
+
+<img src="img/configure_files_server_side.png" width="450" alt="Choose dropdown server-side then new" aria-label="Create new server-side definition" title="Create new server-side file configuration"/>
+
+This will display a new window showing the uploaded files.
+Change the name of the configuration if you want to.
+Select "OK" to complete the configuration and dismiss the window.
+
+<img src="img/configure_files_new_server_side.png" width="450" alt="Finish server-side configuration" aria-label="Click OK to complete configuration" title="Click OK to accept"/>
+
+The configured access is now available to workflows and shown on the page.
+
+<img src="img/configure_files_completed.png" width="450" alt="Configured file storage listing" aria-label="Configured file storage listing" title="Configured file storage listing"/>
+
+You can return to the main page by clicking the "back" button.
+
+<img src="img/configure_files_back.png" width="450" alt="Click back button" aria-label="Click back button for main page" title="Click the back button"/>
+
+### Running a workflow
+
+On the main page of the site, click the `Image` link under **Workflows**.
```

Comment:
calling this 'Image' is confusing, especially since above 'image' refers to a picture as in 'orthomosaic image'. Perhaps call it an algorithm, or a workflow step?

Reply:
I'm trying to keep Image-based workflows separate from Lidar-based ones, etc.

Reply:
ok got it - I thought that this was 'image' as in 'docker image'

I would refer to these as 'RGB Image' as 'Image' could refer to multi/hyperspectral image, xray image, etc

Reply:
I am adding an issue for this clarification: https://github.com/AgPipeline/issues-and-projects/issues/555